### ISSUE TITLE
Missing title member error message needs more details

### ISSUE THREAD

**Issue by StephenNay** [DapperDox/dapperdox#77]
I (too often) see this message when starting DapperDox:

"Error: <method> <resource> references a model definition that does not have a title member."

ANY additional details would be really helpful, such as the name of the model that doesn't have a title member. I have resources that reference one or more models in the parameters, and more models in the responses, and some of those models reference other models, so the error message is telling me that one out of many models doesn't have a title member. Debugging would be much faster if the error message pointed me to the specific model that doesn't have a title member.

**Comment by zxchris** [comment]
Agreed. At the failure point DapperDox does not have access to the model name, due to the way the 3rd party parser folds the parsed spec. We'll have a look and see if there is some other meta-information we can emit at the error to help users track down the issue in their spec.

Thanks for the report!

**Comment by devheroo** [comment]
I am running into the same issue and I am wondering if there is a way to ignore some of these errors ?
it will be great if dapperdox CLI tool accept a parameter to ignore these errors and fill-in missing values with empty or predefined values. please let me know if you have any suggestions in this regard.

**Comment by Sabreclaw** [comment]
+1
